### ISSUE TITLE
Push notification not working on Yun

### ISSUE THREAD

**Issue by abhirocks550** [ParsePlatform/parse-embedded-sdks#60]
I am trying to get push notification on Arduino Yun with Parse SDk version 1.0.3 and Arduino IDE 1.6.5.
The if (Parse.pushAvailable()) {} is not getting executed. I tried sending push from Dashboard as well as from cloud code,none of them worked.

Please help to solve this.

**Comment by abhirocks550** [comment]
I tried creating new account on Parse and with that i tried to send Push notification and it worked.
I had one application where i used push service more than 2 months and suddenly push stopped working on Arduino Yun so i tried to check with new account and application on Parse and it worked.

I am not able to find root cause why it stopped working, is there any limitation for free account push notifications?

Thanks.

**Comment by francip** [comment]
You can see the limitations on the free basic plan at https://parse.com/pricing. Unless you are exceeding 1000000 unique recipients, push notifications should work just fine.

On the other hand, I would suggest checking the installation record for the device on which you are not receiving the push notifications. It's possible the reason you aren't getting the push notifications is due to device configuration, not an issue of plan limits.

**Comment by abhirocks550** [comment]
Device installation is there on Parse.It started working somehow inconsistently, one thing i want to ask i am sending push from cloud code of parse and i am getting that push on my android device consistently but on Arduino yun it is very much inconsistent.

**Comment by francip** [comment]
I'll look into this. Is this happening with a simple app with no business logic, or only with some additional code on the Arduino side?

**Comment by abhirocks550** [comment]
Yes bussiness logic is there. I have written logic in cloud code, a chrone is checking constantly for condition and if that condifition meets then i am sending Push to Arduino as well as Android and IOS.

**Comment by dee101k** [comment]
@francip I am trying to do push notification on esp8266 through parse.But facing similar problem.So can you please suggest some solution for above problem

**Comment by francip** [comment]
@mtl2034 can you take a look?
